shears/mine_for_gold.py

```diff
@@ -36,23 +42,57 @@ def main():
     args = parser.parse_args()
 
     print('Authenticating Reddit API connection...')
+    
+    """Authorized Reddit Instances"""
     reddit = praw.Reddit(
         client_id=args.client_id,
         client_secret=args.client_secret,
         username=_USERNAME,
         password=args.password,
         user_agent=_USER_AGENT,
     )
+
+    """ To verify that you are authenticated as the correct user """
+    print(reddit.user.me())     
+    
     print('Authenticated!')
 
+
     print('Mining for gold...')
     whitelist = get_ticker_set()
-    for comment in reddit.subreddit('wallstreetbets').comments():
-        print(comment.author.name + ' says:\n')
-        print(comment.body)
-        print('Ticker counts: ' + repr(scrape_tickers(comment.body,
-                                                      whitelist=whitelist)))
 
+    subredditName = "wallstreetbets"
```

Comment:
Can we change these double quotes to single quotes for consistency with the rest of the file? Ditto below for the comment JSON keys and the error print statement.